content/ja/docs/concepts/configuration/secret.md

```diff
@@ -752,6 +626,34 @@ echo $SECRET_PASSWORD
 ```
 1f2d1e2e67df
 ```
+// diffの762あたり
```

Comment:
Is this typo?

Reply:
Thanks, This is my mistake. I will fix it